### ISSUE TITLE
1.8 Skin armor not updating

### ISSUE THREAD

**Issue by Noobgalaxies** [Wynncraft/Issues#1563]
So whenever I change my armor, it says "We're drawing your skin, please be patient". However, it never actually changes the skin even after a very long time, and when I put back the armor I used to wear, it suddenly disconnects and reconnects me(You know, that short moment when you change armor?). However, my armor still doesn't change, and now I can't see any armor skins besides the one I'm wearing right now.

TL;DR: It's impossible to change my armor skin.
